app/uploaders/logo_uploader.rb

```diff
@@ -3,10 +3,10 @@
 class LogoUploader < CarrierWave::Uploader::Base
   include Cloudinary::CarrierWave
 
-  process :convert => 'png'
+  process convert: 'png'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.